src/backend/aspen/covidhub_import/implementation.py

```diff
@@ -120,68 +152,196 @@ def import_project(
                 ConsensusGenome.id.desc(),
             )
             .options(
-                joinedload(ConsensusGenome.sample_fastqs)
-                .joinedload(SampleFastqs.czb_id.of_type(DphCZBID))
-                .joinedload(CZBID.genome_submission_info),
+                joinedload(ConsensusGenome.sample_fastqs).joinedload(
+                    SampleFastqs.czb_id
+                )
             )
             .all()
-        )
-        czbid_without_consensus_genomes: Iterable[DphCZBID] = (
-            covidhub_session.query(DphCZBID)
-            .join(Project)
-            .filter(
-                DphCZBID.czb_id.notin_(
-                    {
-                        consensus_genome.sample_fastqs.czb_id.czb_id
-                        for consensus_genome in consensus_genomes
-                    }
-                )
+        }
+
+        internal_czb_ids_metadata: Mapping[str, Tuple[str, datetime.datetime]] = {
+            czbid: (external_accession, collection_date)
+            for czbid, external_accession, collection_date in covidhub_session.query(
+                InternalCZBID.czb_id.label("czb_id"),
+                AccessionSample.accession.label("external_accession"),
```

Comment:
OK! so yes we want this to be

`AccessionSample. location_submitter_id.label("external_accession"),`